x-pack/plugins/ml/public/alerting/config_validator.tsx

```diff
@@ -41,49 +45,82 @@ export const ConfigValidator: FC<ConfigValidatorProps> = React.memo(
+            <EuiCallOut
+              title={
                 <FormattedMessage
-                  id="xpack.ml.alertConditionValidation.alertIntervalTooHighMessage"
-                  defaultMessage="The check interval is greater than the lookback interval. Reduce it to {lookbackInterval} to avoid potentially missing notifications."
-                  values={{
-                    lookbackInterval: alertParams.lookbackInterval,
-                  }}
+                  id="xpack.ml.alertConditionValidation.title"
+                  defaultMessage="Alert condition contains the following issues:"
                 />
-              </li>
-            ) : null}
+              }
+              color="warning"
+              size={'s'}
+            >
+              <ul>
+                {isAlertIntervalTooHigh ? (
+                  <li>
+                    <FormattedMessage
+                      id="xpack.ml.alertConditionValidation.alertIntervalTooHighMessage"
+                      defaultMessage="The check interval is greater than the lookback interval. Reduce it to {lookbackInterval} to avoid potentially missing notifications."
+                      values={{
+                        lookbackInterval: alertParams.lookbackInterval,
+                      }}
+                    />
+                  </li>
+                ) : null}
 
-            {jobWithoutStartedDatafeed.length > 0 ? (
-              <li>
+                {jobWithoutStartedDatafeed.length > 0 ? (
+                  <li>
+                    <FormattedMessage
+                      id="xpack.ml.alertConditionValidation.stoppedDatafeedJobsMessage"
+                      defaultMessage="The datafeed is not started for the following {count, plural, one {job} other {jobs}}: {jobIds}."
+                      values={{
+                        count: jobWithoutStartedDatafeed.length,
+                        jobIds: jobWithoutStartedDatafeed.join(', '),
+                      }}
+                    />
+                  </li>
+                ) : null}
+              </ul>
+            </EuiCallOut>
+            <EuiSpacer size={'m'} />
+          </>
+        ) : null}
+        {notifyWhenWarning ? (
+          <>
+            <EuiCallOut
+              title={
                 <FormattedMessage
-                  id="xpack.ml.alertConditionValidation.stoppedDatafeedJobsMessage"
-                  defaultMessage="The datafeed is not started for the following {count, plural, one {job} other {jobs}}: {jobIds}."
-                  values={{
-                    count: jobWithoutStartedDatafeed.length,
-                    jobIds: jobWithoutStartedDatafeed.join(', '),
-                  }}
+                  id="xpack.ml.alertConditionValidation.notifyWhenWarning"
+                  defaultMessage="Expect to receive duplicate notifications about the same anomaly with a duration of up to {notificationDuration}  {notificationDuration, plural, one {minute} other {minutes}}"
```

Comment:
We can advise both switching back to the default notification behavior or increasing the check interval

Reply:
cc @szabosteve

Reply:
How about something like

```
Expect to receive duplicate notifications about the same anomaly for up to 60 minutes. Consider increasing the check interval or notifying only on status change.
```

Reply:
What do you think about the following?
```
Expect to receive duplicate notifications about the same anomaly for up to 60 minutes. Increase the check interval or switch to notify only on status change to avoid duplicate notifications.
```
It's a bit longer but the suggestion in the second sentence is a bit more direct, too.